judge/models/contest.py

```diff
@@ -316,14 +346,16 @@ def is_accessible_by(self, user):
         else:
             return True
 
+    def is_editor(self, profile):
+        return (self.authors.filter(id=profile.id) | self.curators.filter(id=profile.id)).exists()
+
     def is_editable_by(self, user):
         # If the user can edit all contests
         if user.has_perm('judge.edit_all_contest'):
             return True
 
-        # If the user is a contest organizer
-        if user.has_perm('judge.edit_own_contest') and \
-                self.organizers.filter(id=user.profile.id).exists():
+        # If the user is a contest organizer or curator
+        if user.has_perm('judge.edit_own_contest') and user.profile.id in self.editor_ids:
```

Comment:
What's the justification for using `in self.editor_ids` here and `is_editor` in `can_see_full_scoreboard` and `access_check`?

Reply:
Changed both to use `editor_ids`.